templates/docker-compose.j2

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+services:
+    # web:
+    #   image: nginx:latest
+    #   ports:
+    #     - "8080:80"
+    #   volumes:
+    #     - /opt/nginx/html:/usr/share/nginx/html:ro
+    #     - /opt/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+  cache:
+    image: nginx:latest
+    ports:
+      - "127.0.0.1:80:80"
```

Comment:
In my testing I confirmed it was only listening locally. Were you able to do an NMAP scan or similar to help verify this?